libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ApolloResponse.kt

```diff
@@ -28,11 +28,23 @@ private constructor(
 
     /**
      * GraphQL [operation] execution errors returned by the server to let client know that something has gone wrong.
-     * This can either be null or empty depending on what your server sends back
+     * This can either be null or empty depending on what your server sends back.
+     *
+     * See also [exception] for exceptions happening before a valid GraphQL response could be received.
```

Comment:
Re-reading this. The GraphQL errors are not only from execution. Server validation errors will also appear here (although they should "never happen" because we also validate client side). Also feels like this should mention partial data. 

Tentative change:

```
[GraphQL errors](https://spec.graphql.org/October2021/#sec-Errors) returned by the server to let client know that something has gone wrong.

If no GraphQL error was raised, [errors] is null. Else it's a non-empty list of errors indicating where the error(s) happened.

Note that because GraphQL allows partial data, it is possible to have both [data] non null and [errors] non null.

See also [exception] for exceptions happening before a valid GraphQL response could be received.
```